com.github.maoschanz.DynamicWallpaperEditor.json

```diff
@@ -13,12 +13,17 @@
 	"modules" : [{
 		"name" : "dynamic-wallpaper-editor",
 		"buildsystem" : "meson",
-		"sources" : [{
-			"type" : "git",
-			"tag" : "2.7",
-			"commit" : "1973696f339673ad31b753cc50bc3c65c4fc66a7",
-			"url" : "https://github.com/maoschanz/dynamic-wallpaper-editor.git"
-		}]
+		"sources" : [
+			{
+				"type" : "git",
+				"url" : "https://github.com/maoschanz/dynamic-wallpaper-editor.git",
+				"commit" : "6bf092c01b47f5fc4cce06f5f3869df5bb7bfc3c"
```

Comment:
Why is it now a commit? Please try to keep it to the last release tag to avoid introducing regressions.

Reply:
Thanks.